code/_onclick/hud/screen_objects.dm

```diff
@@ -90,107 +90,6 @@
-		return
-	var/mob/living/carbon/C = usr
-	if(C.incapacitated())
-		return
-
-	if(C.internal)
-		C.internal = null
-		to_chat(C, "<span class='notice'>No longer running on internals.</span>")
-		icon_state = "internal0"
-	else
-		var/no_mask = FALSE
-		if(!C.get_organ_slot("breathing_tube"))
-			if(!C.wear_mask || !(C.wear_mask.flags & AIRTIGHT))
-				if(ishuman(C))
-					var/mob/living/carbon/human/H = C
-					if(!H.head || !(H.head.flags & AIRTIGHT))
-						no_mask = TRUE
-
-		if(no_mask)
-			to_chat(C, "<span class='notice'>You are not wearing a suitable mask or helmet.</span>")
-			return
-
-		var/list/nicename = null
-		var/list/tankcheck = null
-		var/breathes = "oxygen"
-		var/list/contents = list()
-		var/from = "on"
-
-		if(ishuman(C))
-			var/mob/living/carbon/human/H = C
-			breathes = H.species.breath_type
-			nicename = list("suit", "back", "belt", "right hand", "left hand", "left pocket", "right pocket")
-			tankcheck = list(H.s_store, C.back, H.belt, C.r_hand, C.l_hand, H.l_store, H.r_store)
-		else
-			nicename = list("right hand", "left hand", "back")
-			tankcheck = list(C.r_hand, C.l_hand, C.back)
-
-		// Rigs are a fucking pain since they keep an air tank in nullspace.
-		if(istype(C.back,/obj/item/weapon/rig))
-			var/obj/item/weapon/rig/rig = C.back
-			if(rig.air_supply)
-				from = "in"
-				nicename |= "hardsuit"
-				tankcheck |= rig.air_supply
-
-		for(var/i = 1, i < tankcheck.len + 1, ++i)
-			if(istype(tankcheck[i], /obj/item/weapon/tank))
-				var/obj/item/weapon/tank/t = tankcheck[i]
-				switch(breathes)
-					if("nitrogen")
-						if(t.air_contents.nitrogen && !t.air_contents.oxygen)
-							contents.Add(t.air_contents.nitrogen)
-						else
-							contents.Add(0)
-					if("oxygen")
-						if(t.air_contents.oxygen && !t.air_contents.toxins)
-							contents.Add(t.air_contents.oxygen)
-						else
-							contents.Add(0)
-					if("carbon dioxide")
-						if(t.air_contents.carbon_dioxide && !t.air_contents.toxins)
-							contents.Add(t.air_contents.carbon_dioxide)
-						else
-							contents.Add(0)
-					if("plasma")
-						if(t.air_contents.toxins)
-							contents.Add(t.air_contents.toxins)
-						else
-							contents.Add(0)
-			else
-				//no tank so we set contents to 0
-				contents.Add(0)
-
-		//Alright now we know the contents of the tanks so we have to pick the best one.
-		var/best = 0
-		var/bestcontents = 0
-		for(var/i=1, i <  contents.len + 1 , ++i)
-			if(!contents[i])
-				continue
-			if(contents[i] > bestcontents)
-				best = i
-				bestcontents = contents[i]
-		//We've determined the best container now we set it as our internals
-		if(best)
-			to_chat(C, "<span class='notice'>You are now running on internals from [tankcheck[best]] [from] your [nicename[best]].</span>")
-			C.internal = tankcheck[best]
-
-		if(C.internal)
-			icon_state = "internal1"
-		else
-			to_chat(C, "<span class='notice'>You don't have a[breathes == "oxygen" ? "n oxygen" : addtext(" ",breathes)] tank.</span>")
```

Comment:
Where-in Baystation decided it was best to make it so that you couldn't turn on internals if you didn't breath the gas inside the internals.

*slowclap*